### PR TITLE
Adding rabbitmq with the management module

### PR DESCRIPTION
**Pull request by hush-hush**

### CODE REVIEW

File: rabbitmq/auto_conf/rabbitmq.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+docker_images:
+  - rabbitmq
```
Comment (xvello) [inline rabbitmq/auto_conf/rabbitmq.yaml]: As we are adding test data, please give the sd ID a more specific name (as in `rabbitmq-testqueues`) to differenciate from other flavors that could be added later

File: rabbitmq/auto_conf/rabbitmq.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    # Use the `queues` or `queues_regexes` parameters to specify the queues you'd like to
+    # collect metrics on (up to 200 queues).
+    # If you have less than 200 queues, you don't have to set this parameter,
+    # the metrics will be collected on all the queues by default.
+    # If you have set up vhosts, set the queue names as `vhost_name/queue_name`.
+    # If you have `tag_families` enabled, the first captured group in the regex
+    # will be used as the queue_family tag
+    #
+    # queues:
+    #   - queue1
```
Comment (xvello) [inline rabbitmq/auto_conf/rabbitmq.yaml:54]: Should these be customized?

**Review by xvello — CHANGES_REQUESTED** [review]
Great addition, just missing config (monitor the test queues we created)

File: rabbitmq/auto_conf/rabbitmq.yaml (hunk 2 above)
Comment (hush-hush) [reply rabbitmq/auto_conf/rabbitmq.yaml:54]: Indeed, but let's use the queues_regexes.

**Review by xvello — APPROVED** [review]
🚢 it!

**Pull request merged**